src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -641,6 +641,11 @@ export class ExtensionsListView extends ViewletPanel {
 	static isKeymapsRecommendedExtensionsQuery(query: string): boolean {
 		return /@recommended:keymaps/i.test(query);
 	}
+
+	focus(): void {
+		super.focus();
+		this.list.domFocus();
```

Comment:
I suggest focusing the first list element right here, on top of giving it DOM focus.

Reply:
Yeah 👍

Reply:
Focused first element in the `onDidFocus` so that it also runs on tab-focus (which doesn't trigger `focus()`)